### PR TITLE
Fix a couple of typos and update for modern local::lib variables

### PR DESCRIPTION
**Pull request by sambrightman**

### CODE REVIEW

File: penv.pl [hunk 1]
```diff
@@ -85,4 +87,4 @@
 if [ -n "$BASH" -o -n "$ZSH_VERSION" ] ; then
     hash -r
 fi
-eval $(perl -I{{ PENV }}/perl5/lib/perl5 -Mlocal::lib={{ PENV }})
+eval $(perl -I{{ PENV }}/lib/perl5 -Mlocal::lib={{ PENV }})
```
Comment (sambrightman) [inline penv.pl]: This is maybe not a typo, but due to bootstrapping `local::lib` from home directory? Still looks a bit odd.

**Comment by jtopjian** [comment]
Thank you for this :)

To be honest, I haven't used penv for several years. I'm happy to merge this PR, so let me know when it's ready or if there are any other changes you'd like to make.

**Comment by sambrightman** [comment]
Great! Previously I had `local::lib` installed at system level for my own use only, so the incorrect `-I` was irrelevant to me. With the most recent push I have it using bootstrapped `local::lib` on Travis for a wrapper project, so I believe now correct (the point of the `-I` is to support the default bootstrap location from what I can tell).

**Comment by jtopjian** [comment]
Sounds good to me. Thank you again!

**Pull request merged**